GoToProject/src/components/SelectImagesActions/index.js

```diff
@@ -0,0 +1,23 @@
+import {Alert} from 'react-native';
+import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
+export const OpenGaleria = (newImage, doAction) => {
+  let action = '';
+  const options = { StorageOptions: {path: 'image', mediaType: 'photo', }, includeBase64: true, };
+  if (doAction == true) {
+    action = launchImageLibrary;
+  } else {
+    action = launchCamera;
+  }
```

Comment:
You can use ternary operator `condition ? value_if_true : value_if_false`

Reply:
Done, thanks for your comment